server/acl_casbin_policy_dba.csv

```diff
@@ -50,6 +50,7 @@ p, DBA, /instance/{instanceID}/role, POST
 p, DBA, /instance/{instanceID}/role/{roleName}, GET
 p, DBA, /instance/{instanceID}/role/{roleName}, PATCH
 p, DBA, /instance/{instanceID}/role/{roleName}, DELETE
+p, DBA, /instance/{instanceID}/database/{database}, PATCH
```

Comment:
Use `instanceName` instead? Same for `server/acl_casbin_policy_owner.csv`

Reply:
Done.